### PR TITLE
Fix test_vm_migrate for RHV

### PR DESCRIPTION
**Pull request by bargulg**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9ae20dacfd812e9355f00ded96a37857e73c7d43:

`cfme/infrastructure/virtual_machines.py`:
- :bangbang: Line 252:47: **E202** *whitespace before ']'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by bargulg** [comment]
If it's failing on "RequestException: Multiple requests with matching content found - be more specific!", that's caused by old vm_migrate requests which were not deleted

File: cfme/infrastructure/virtual_machines.py [hunk 1]
```diff
@@ -247,8 +247,9 @@ def migrate_vm(self, email=None, first_name=None, last_name=None,
             "last_name": last_name,
             "email": email,
             "host_name": {"name": prov_data.get("host")},
-            "datastore_name": {"name": prov_data.get("datastore")}
         }
+        if not self.provider.type == 'rhevm':
```
Comment (izapolsk) [inline cfme/infrastructure/virtual_machines.py]: @OPTIONAL: I guess provider.one_of would be better here

Comment (izapolsk) [inline cfme/infrastructure/virtual_machines.py]: @REQUIRED: please use correct constructions like != instead of not ==
readability counts (c)

**Comment by mfalesni** [comment]
@bargulg ping, any progress on this PR?

**Comment by bargulg** [comment]
I didn't have enough time to work on this, because of manual bug verification and other stuff...
Anyway, this is failing on PRT only because there are other providers in PRT. This won't happen on RHV jenkins, so we should try running PRT separately for different providers if possible.
I will fix it to use provider.one_of soon, when i have enough time to fix and verify it.

File: cfme/infrastructure/virtual_machines.py (hunk 1 above)
Comment (bargulg) [inline cfme/infrastructure/virtual_machines.py]: done

Comment (bargulg) [inline cfme/infrastructure/virtual_machines.py]: done

**Comment by bargulg** [comment]
verified locally and on RHV jenkins

**Comment by dajoRH** [comment]
Lint report for commit 0dea08f6dbc66c8a82b85d3fb8eed309e3e78df3:

`cfme/tests/infrastructure/test_vm_migrate.py`:
- :red_circle: Line 2:1: **F401** *'fauxfactory' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by mfalesni** [comment]
It was skipping on our RHEV because of our RHEV being angry currently, but it did not break the vsphere test and @bargulg said it ran well on their RHEV. Therefore merging.

**Pull request merged**